static/gsAdmin/routes.tsx

```diff
@@ -40,101 +40,252 @@ import UserDetails from 'admin/views/userDetails';
-        <Route path=":broadcastId/" component={BroadcastDetails} />
-      </Route>
-
-      <Route path="customers/">
-        <IndexRoute component={Customers} />
-        <Route path=":orgId/">
-          <IndexRoute component={CustomerDetails} />
-          <Route path="upgrade-request/" component={CustomerUpgradeRequest} />
-          <Route path="projects/:projectId/" component={ProjectDetails} />
-          <Route path="invoices/:region/:invoiceId/" component={InvoiceDetails} />
-        </Route>
-      </Route>
-
-      <Route path="doc-integrations/">
-        <IndexRoute component={DocIntegrations} />
-        <Route path=":docIntegrationSlug/" component={DocIntegrationDetails} />
-      </Route>
-      <Route path="debugging-tools/">
-        <IndexRoute component={DebuggingTools} />
-      </Route>
-      <Route path="policies/">
-        <IndexRoute component={Policies} />
-        <Route path=":policySlug" component={PolicyDetails} />
-      </Route>
-
-      <Route path="private-apis/">
-        <IndexRoute component={PrivateAPIs} />
-      </Route>
-
-      <Route path="relocations/">
-        <IndexRoute component={Relocations} />
-        <Route path="new/" component={RelocationCreate} />
-        <Route path=":regionName/:relocationUuid/" component={RelocationDetails} />
-        <Route
-          path=":regionName/:relocationUuid/:artifactKind/:fileName/"
-          component={RelocationArtifactDetails}
-        />
-      </Route>
-
-      <Route path="employees/">
-        <IndexRoute component={SentryEmployees} />
-      </Route>
-
-      <Route path="promocodes/">
-        <IndexRoute component={PromoCodes} />
-        <Route path=":codeId/" component={PromoCodeDetails} />
-      </Route>
-
-      <Route path="sentry-apps/">
-        <IndexRoute component={SentryApps} />
-        <Route path=":sentryAppSlug/" component={SentryAppDetails} />
-      </Route>
-
-      <Route path="users/">
-        <IndexRoute component={Users} />
-        <Route path=":userId/" component={UserDetails} />
-      </Route>
-
-      <Route path="options/">
-        <IndexRoute component={Options} />
-      </Route>
-      <Route path="data-requests/" component={DataRequests} />
-      <Route path="billingadmins/" component={BillingAdmins} />
-
-      <Route path="invoices/">
-        <IndexRoute component={Invoices} />
-        <Route path=":invoiceId/" component={InvoiceDetails} />
-      </Route>
-
-      <Route path="instance-level-oauth">
-        <IndexRoute component={InstanceLevelOAuth} />
-        <Route path=":clientID/" component={InstanceLevelOAuthDetails} />
-      </Route>
-
-      <Route path="billing-plans/">
-        <IndexRoute component={BillingPlans} />
-      </Route>
-
-      <Route path="spike-projection-generation/">
-        <IndexRoute component={GenerateSpikeProjectionsForBatch} />
-      </Route>
-      <Route path="*" component={NotFound} />
-    </Route>
-  );
+  return [translateSentryRoute(routes)];
```

Comment:
is there an extra `[]` wrapping things here? hard to tell

Reply:
It actually needs to return a `RouteObject[]`, since that's what this takes https://docs.sentry.io/platforms/javascript/guides/react/features/react-router/v6/